docs/dev_install.rst

```diff
@@ -28,6 +28,7 @@ Installing Depedencies
 
   .. code-block:: bash
 
+    sudo pacman -S ncurses5-compat-libs  # /usr/lib/libtinfo.so.5
```

Comment:
Actually this is an package in only `archlinuxcn`:
```
[bate@archit gpu]$ sudo pacman -Ss libtinfo5
archlinuxcn/ncurses5-compat-libs 6.1-1
    System V Release 4.0 curses emulation library, ABI 5
```
Should I tell them to add archlinuxcn source to their `/etc/pacman.conf` to install this dep, even if they're not chinese at all?

Reply:
Fixed by using `yaourt`, install from aur/ncurse5-compat-libs.